### DISCUSSION TITLE
Github support not responding after over a week

### DISCUSSION THREAD

**Discussion by fpapalardo** [community/community#34840] [Other Feature Feedback, Questions, & Ideas]
I've open a github support ticket about 9 days ago and I got no response so far. Any idea why? Is there any estimated response time per ticket open to them?

**Comment by jfullstackdev** [comment]
posting it here adds more visibility but still there is no assurance, 

but I have seen so many posts about their tickets being responded here by the Staff members,

**Reply by fpapalardo** [reply]
Hopefully helps then!

**Comment by radar-blip** [comment]
GitHub Support responds to tickets on a first come first serve basis. 
As far as an estimated response time, I dont think they can guarantee anything there. 
Also, Ive seen it mentioned that it doesn't help to submit multiple tickets, that would actually just slow things down more.
I know this isn't what you want to hear, but everything I mentioned is from previous posts from Staff.

Anyway, I figure they probably have their hands full. Hang in there!

**Reply by fpapalardo** [reply]
Yeah I was not planning on adding another ticket. Just keep waiting sadly. Hopefully is not a long wait!

**Comment by chrisstef** [comment]
I am disappointed. It's been 1 month after my account was flagged as spam, for no obvious reason, and this has already costed me 2 job applications where I couldn't showcase my codebase and give access to a 3rd party app required for the application process.

**Comment by Junaid1240651** [comment]
Hi GitHub Team and community members,

I want support regarding my GitHub account as it's been flagged for the past more then 1 week, I have opened an ticket for this but still no response.
Please help me get this resolved as soon as possible 🥺🥺.
The following is the ticket number:
Ticket - #2718458
username: Junaid1240651

**Comment by baharbaba** [comment]
I've been waiting for a response for 1 month. A week ago, I received a message saying "We have not forgotten about you..." but now I feel like I have been forgotten. From now on, I'll be tagging @github-staff in every discussion that has this issue. I guess you need to be Karen to get a problem solved.
